docs: Describe token locality for auth methods (#12019)
hashicorp/consul: website/content/docs/security/acl/auth-methods/oidc.mdx

  contain sensitive information, this option should not be used in production.

@include 'jwt_claim_mapping_details.mdx'

## Token Locality

> By default, tokens created by the OIDC Auth Method will have local scope and will
> therefore only be valid for the datacenter in which they were created. When the

Reviewer: ```suggestion
By default, tokens created by the OIDC Auth Method are scoped to the local datacenter in which they were created. When the
```
same as the suggestion in the JWT section

Auth Method is created with [`-token-locality=global`](/commands/acl/auth-method/create#token-locality),

> all tokens will be replicated to all other datacenters. Cross-datacenter requests

Reviewer: ```suggestion
all tokens will be replicated to all other datacenters. Cross-datacenter requests
```
For clarity, future perfect tense is acceptable in this sentence because the action actually will take place later during the replication process.
